source/configuration_manual/howto/dovecot_lda_sendmail.rst

```diff
@@ -0,0 +1,108 @@
+.. _howto-dovecot_lda_sendmail:
+
+Dovecot LDA with Sendmail
+=========================
+
+The following describes how to configure Sendmail to use ``dovecot-lda``
+where ``root`` permission is not granted and Dovecot runs under a single
+user ID. It may need some adjustment for more typical setups. Other
+assumptions are that Sendmail is configured for virtual hosting and that
+local-system mail delivery is not handled by ``dovecot-lda``.
+
+Allowing that ``sendmail.mc`` has ``MAILER(procmail)dnl`` included, edit
+``sendmail.cf`` adding these lines after the ``Mprocmail`` definition:
+
+::
+
+   ######################*****##############
+   ###   DOVECOT Mailer specification                              ###
+   ##################*****##################
+   Mdovecot,   P=/usr/local/libexec/dovecot/dovecot-lda, F=DFMPhnu9,
+                    S=EnvFromSMTP/HdrFromSMTP, R=EnvToSMTP/HdrFromSMTP,
+                    T=DNS/RFC822/X-Unix,
+                    A=/usr/local/libexec/dovecot/dovecot-lda -d $u
```

Comment:
This looks improperly formatted, but might just be my aesthetics. :shrug:

Reply:
The header comment is still having a few more extraneous spaces before the closing `###`.
I.e. replace with:
```
######################*****##############
###   DOVECOT Mailer specification    ###
##################*****##################
```